calliope/backend/pyomo/constraints/modelwide.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright (C) 2013-2019 Calliope contributors listed in AUTHORS.
+Licensed under the Apache 2.0 License (see LICENSE file).
+
+modelwide.py
+~~~~~~~~~
+
+Model-wide constraints.
+
+"""
+
+import numpy as np
+import pyomo.core as po  # pylint: disable=import-error
+
+
+def load_constraints(backend_model):
+    model_data_dict = backend_model.__calliope_model_data__['data']
+
+    if 'modelwide_demand_share_min' in model_data_dict:
+        backend_model.modelwide_demand_share_min_constraint = po.Constraint(
+            backend_model.modelwide_constraint_groups,
+            backend_model.carriers,
+            ['min'], rule=demand_share_constraint_rule
+        )
+    if 'modelwide_demand_share_max' in model_data_dict:
+        backend_model.modelwide_demand_share_max_constraint = po.Constraint(
+            backend_model.modelwide_constraint_groups,
+            backend_model.carriers,
+            ['max'], rule=demand_share_constraint_rule
+        )
+
+
+def equalizer(lhs, rhs, sign):
```

Comment:
We have this equalizer (equaliser?) elsewhere, is it worth making it a util function?

Reply:
Only once instance will remain once the deprecated group_share constraints are removed, so can stay as is for now